### DISCUSSION TITLE
Request to add ThisVid.com to site list

### DISCUSSION THREAD

**Discussion by leadsteelcopper** [AAndyProgram/SCrawler#95] [Ideas]
Hello,

I would like to request ThisVid.com be added to as a future site option. Hopefully this can be added in the future.

**Comment by AAndyProgram** [comment]
You are not alone. At least 40 people (on Discord, Matrix, Reddit) have asked me to add ThisVid. Unfortunately, I haven't found an easy way to do this. This site only generates a video download request after the user clicks the Play button. I tried to download a video using Selenium, but it has its own complexity, it uses a browser (internal and hidden, but still), and I'm not sure this is the best way.

I'm currently very busy at work and I hope that this month I will release only bug fixes (RedGifs, TikTok, etc.), but nothing more. I will return to development of this site downloader only in January.

**Comment by AAndyProgram** [comment]
**Added**. Please update to the [latest release](https://github.com/AAndyProgram/SCrawler/releases/latest).
